### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by PollyThePancake** [AllTheMods/ATM-10#984]
### Possible Fixes

Yes

### Modpack Version

1.12

### What happened?

The latest updated added Infernal Mobs. This is a neat addition but now mob spawner farms are also spawning them, and lots of the infernal mobs are teleporting out of the spawners to random areas. This has caused mobs to scatter all around my base both underground and on the surface, and i resorted to using commands to remove them all and rendered my mob spawner farms unusable. Please remove the spawners ability to spawn Infernal Mobs or remove Infernal Mobs entirely

**Comment by Yamijiro** [comment]
Try using ender inhibitors from mob grinding utils.

**Comment by BurnSaibot** [comment]
Ender Inhibitors seems to not stop them from teleporting :/

**Comment by K1kk0m4n** [comment]
Yeah got same problem. Cant really use mob farm until fixed and killing those mobs manually are pain

**Comment by kman3107** [comment]
I had a lot of "fun" last night, because of this. Was going to bed and leaving game on. I turn on the mob spawner and finish my enchanting setup. I randomly check on the farm before logging off and see what looked to be like a hundred different mobs spread all around the base of my base.. My base has ender inhibitors placed strategically to avoid those pesky endermen from placing random dirt blocks all over the place.

First I tried killing them with my sword.. That was a hassle, so I went around with a Mob Crusher and had the same issue as spocane, with loot and xp dropping in the world (not that I cared, since all I wanted was to get done and go to sleep).

PS: Added my sitch here since there isn't a point in starting another bug post, and there is no upvote for issues on Github.
